site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -204,16 +207,30 @@ class AlgoliaUpdater {
   uploadToAlgolia = async ({ language, entries }) => {
     const indexName = `instant_search-${language}`;
 
+    const featuredReplicaIndexName = indexName + '-featured';
+
     const index = this.algoliaClient.initIndex(indexName);
 
     await index.saveObjects(entries);
 
-    await index.setSettings({
-      ...algoliaSettings,
-      attributeForDistinct: 'incident_id',
-      indexLanguages: [language],
-      queryLanguages: [language],
-    });
+    index
+      .setSettings({
+        ...algoliaSettings,
+        attributeForDistinct: 'incident_id',
+        indexLanguages: [language],
+        queryLanguages: [language],
+        replicas: [featuredReplicaIndexName],
+      })
+      .catch((e) => console.error(e))
+      .then(() => {
+        const featuredReplicaIndex = this.algoliaClient.initIndex(featuredReplicaIndexName);
+
+        featuredReplicaIndex
+          .setSettings({
+            ranking: ['desc(featured)', 'desc(text)'],
+          })
+          .catch((e) => console.error(e));
```

Comment:
And this one. 

It is preferable that the build crashes and nothing gets deployed than deploying the app with some indexes working and some broken.